socorro/external/boto/connection_context.py

```diff
@@ -154,15 +150,15 @@ def build_client(self):
         wait_time_generator=wait_times_access,
         module_logger=logger,
     )
-    def save_file(self, path, data):
+    def save_file(self, bucket, path, data):
```

Comment:
I changed the role of this connection context so it just wraps the boto API and handles the connection specifically. It no longer builds keys, or knows about buckets, or any of that.